hack/run_in_temp.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
Lev would like to know why you have a HACK in this code.

Reply:
The `hack` folder does already exists in the oadp-operator codebase:
   https://github.com/openshift/oadp-operator/tree/master/hack
   
And it seems natural to put scripts within one folder that may have any name.

However in operator-sdk go-based operators it's a common to use hack/ folder for utility scripts as described in the:
https://sdk.operatorframework.io/docs/overview/project-layout/#golang

We use an absolute path of the place where the script resides, but we need to go one level up to be able to copy/rsync folder structure of the main oadp-operator, that is why we have on line number 4 comment about that.

Reply:
@levhayutin FYI.. thank you @mpryc :)